tests/nodes/start.sh

```diff
@@ -38,7 +38,9 @@ cargo build
 cd "$nodes_dir" || exit 1
 
 start() {
-    ../../target/debug/fnn "$@"
+    log_file="${2}.log"
```

Comment:
this is debugging code?

Reply:
Yes, E2E tests will upload nodes log files on failure, we can merge the code into develop for debugging e2e tests.